task_handler_layer/task_buoy/msg/ResizedImage.msg

```diff
@@ -0,0 +1,3 @@
+uint32 original_height
```

Comment:
All this is added by Soumye. It was not there originally in my code. So maybe it is necessary for image transport. You can ask him about it